ctru-rs/src/services/gfx.rs

```diff
@@ -55,6 +55,28 @@ pub trait Screen: private::Sealed {
         unsafe { ctru_sys::gfxSetDoubleBuffering(self.as_raw(), enabled) }
     }
 
+    /// Swaps the video buffers.
+    ///
+    /// This should be used even if double buffering is disabled.
+    fn flush_buffer(&mut self) {
+        let framebuffer = self.raw_framebuffer();
+
+        // Flush the data array. `self.raw_framebuffer` should get the correct parameters for all kinds of screens
+        unsafe {
+            ctru_sys::GSPGPU_FlushDataCache(
+                framebuffer.ptr.cast(),
+                (framebuffer.height * framebuffer.width) as u32,
+            )
+        };
+    }
+
+    /// Swaps the video buffers.
+    ///
+    /// This should be used even if double buffering is disabled.
+    fn swap_buffers(&mut self) {
+        unsafe { ctru_sys::gfxScreenSwapBuffers(self.side().into(), true) };
```

Comment:
ah nice, I was wondering how these would work since the `gfxFlushBudders` etc were global, I didn't realize there are APIs for doing it per screen.

Should we set the `hasStereo` parameter to `false` here by default, and set to `true` only if the screen in question is the "combined" top screen?